rules/test.bzl

```diff
@@ -160,3 +160,14 @@ def ios_ui_test(name, apple_library = apple_library, **kwargs):
     if not kwargs.get("test_host", None):
         fail("test_host is required for ios_ui_test.")
     _ios_test(name, rules_apple_ios_ui_test, rules_apple_ios_ui_test_suite, apple_library, **kwargs)
+
+def ios_unit_snapshot_test(name, apple_library = apple_library, **kwargs):
+    """
+    Builds and packages iOS Unit Snapshot Tests.
+
+    Args:
+        name: The name of the UI test.
+        apple_library: The macro used to package sources into a library.
+        **kwargs: Arguments passed to the apple_library and ios_ui_test rules as appropriate.
+    """
+    _ios_test(name, rules_apple_ios_ui_test, rules_apple_ios_ui_test_suite, apple_library, internal_test_deps = ["@bazel_tools//tools/cpp/runfiles"], **kwargs)
```

Comment:
Should this be `rules_apple_ios_unit_test`?

Reply:
Good catch 👍